lib/bolt/bolt_option_parser.rb

```diff
@@ -467,7 +476,7 @@ module            Manage Bolt project modules
           new
       
       USAGE
-          bolt plan new <plan> [options]
+          bolt plan new NAME [OPTIONS]
```

Comment:
Hm, to me 'plan' is more clear than 'name', but 'plan name' is probably most clear.